catalyst/utils/mixup.py

```diff
@@ -0,0 +1,47 @@
+from typing import Dict, Iterable
+
+import numpy as np
+import torch
+
+
+def mixup_batch(
+    batch: Dict[str, torch.Tensor], keys: Iterable[str], alpha: float = 0.2, mode: str = "replace"
```

Comment:
speaking of API, why should we pass `batch: Dict[str, torch.Tensor], keys: Iterable[str]` if we could pass just `batch: Tuple[torch.Tensor]`?

I mean, it's much easier to prepare the batch for mixup outside of mixup function, isn't it?
like 
```python
keys = [str]
batch = {str: tensor}
mixup_batch = [batch[key] for key in keys]
mixup_batch = mixup_batch(mixup_batch, alpha = 0.2, mode = "replace")
```

Reply:
Are we considering someone else to use this utility? If yes, then it seems that it is better to let her do everything(include preprocessing batch).

Reply:
it makes the function complex, let it been as simple as possible and move all the preprocessing logic outside of the function